amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2495,7 +2497,194 @@ paths:
+        - in: body
+          name: nodeBodyCreate
+          description: |
+            The node information to create.
+          schema:
+            $ref: '#/definitions/RetentionNodeBodyCreate'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/RetentionScheduleResponse'
+        '400':
+          description: |
+            Invalid parameter: recordCategoryId is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to create retention schedule
+        '404':
+          description: recordCategoryId does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    get:
+      tags:
+        - retention-schedule
+      summary: Get a list of retention schedules
+      description: |
+        Get a list of retention schedules.
+        
+        You can use the **include** parameter (include=actions) to return additional information.
 
+      operationId: getRetentionScheduleList
+      parameters:
+        - $ref: '#/parameters/recordCategoryIdParam'
+        - $ref: '#/parameters/retentionScheduleIncludeParam'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/RetentionScheduleResponseList'
+        '400':
+          description: |
+            Invalid parameter: recordCategoryId is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to get retention schedule
+        '404':
+          description: recordCategoryId does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/retention-schedules/{id}/retention-steps':
+    post:
+      tags:
+        - retention-schedule
+      summary: Create a step in the retention schedule
+      description: |
+        Create a step in the retention schedule.
+        List of Steps:
+          * **retain**
+          * **cutoff**
+          * **accession**
+          * **transfer**
+          * **destroy**
+        
+        Order of steps:
+          * "retain" or "cutoff" should be first
+          * can't use "cutoff" after "transfer" or "accession"
+          * "transfer" action is allowed multiple times
+          * "destroy" must be the last action
+        
+        **combineDispositionStepConditions** property is required only for **Accession** step and its used to specify where to combine the period condition and events for the step execution or only consider one of them.
```

Comment:
Should it be `**Accession**` or `**accession**` (as above)?

Reply:
I think most of this should be a description on the actual property below. Perhaps the following split:

Here:
```
The **combineDispositionStepConditions** property is only valid for an **accession** step.
```

On property:
```
Used to specify whether to combine the period condition and events for the step execution or only consider one of them.
```

The description I've written here is not sufficient IMO. It doesn't tell me how to use this property.

Reply:
The above property of the request body. Its not query parameter or path parameter hence not sure how can we define the description on the property. So we added in the description. we also updated the description.

Reply:
Please do the same thing in the other places I mentioned too.

Reply:
Yes, added a description to the definition in the body.